MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -862,6 +876,87 @@ public Task ConvertImage(string inputPath, string outputPath)
             throw new NotImplementedException();
         }
 
+        /// <inheritdoc />
+        public IEnumerable<string> GetPrimaryPlaylistVobFiles(string path, uint? titleNumber)
+        {
+            // Minimum size 450 MB
+            const long MinPlayableSize = 471859200;
+
+            // Try to eliminate menus and intros by skipping all files at the front of the list that are less than the minimum size
+            // Once we reach a file that is at least the minimum, return all subsequent ones
+            var allVobs = _fileSystem.GetFiles(path, true)
+                .Where(file => string.Equals(file.Extension, ".vob", StringComparison.OrdinalIgnoreCase))
+                .Where(file => !string.Equals(file.Name, "VIDEO_TS.VOB", StringComparison.OrdinalIgnoreCase))
+                .Where(file => !file.Name.EndsWith("_0.vob", StringComparison.OrdinalIgnoreCase))
+                .OrderBy(i => i.FullName)
+                .ToList();
+
+            // If we didn't find any satisfying the min length, just take them all
+            if (allVobs.Count == 0)
+            {
+                _logger.LogWarning("No vobs found in dvd structure.");
+                return Enumerable.Empty<string>();
+            }
+
+            if (titleNumber.HasValue)
+            {
+                var prefix = string.Format(
+                    CultureInfo.InvariantCulture,
+                    titleNumber.Value >= 10 ? "VTS_{0}_" : "VTS_0{0}_",
```

Comment:
Isn't that what padding is for? E.g. `VTS_{0:D2}_`. Or can it have more than 99?